data/yam/agama/auto/sle_default_no_reg_ppc64le.jsonnet

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
actually is `sles` not `sle`, feel free to rename any other file you find with wrong prefix.
Use full words when possible, so it could be: `sles_default_no_registration_ppc64le.jsonnet`.

Reply:
Updated the names, as well as all the other files named as `sle_`